db/seeds/custom_contents.rb

```diff
@@ -41,7 +41,11 @@
   {key: Memberships::TerminateSacMembershipMailer::CONFIRMATION,
    placeholders_required: "person-name, sektion-name, terminate-on"},
   {key: Memberships::SwitchStammsektionMailer::CONFIRMATION,
-   placeholders_required: "person-name, group-name"})
+   placeholders_required: "person-name, group-name"},
```

Comment:
wir sollten hier `sektion-name` statt `group-name` verwende, damit eindeutigt ist, was hier erwartet wird / gesetzt werden soll (passes in der spez an)